projects/wave-core-new/src/lib/util/directives/autocomplete-select.directive.ts

```diff
@@ -0,0 +1,55 @@
+import {Directive, Input, OnChanges, OnDestroy, SimpleChanges} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {Observable, ReplaySubject, Subscription} from 'rxjs';
+
+@Directive({
+    selector: '[waveAutocompleteSelect]',
+    exportAs: 'waveAutocompleteSelect',
+})
+export class AutocompleteSelectDirective<T> implements OnChanges, OnDestroy {
+    @Input('waveAutocompleteSelectAllValues') allValues: Array<T> = [];
+    @Input('waveAutocompleteSelectSearchPredicate') searchPredicate: AutocompleteSelectPredicateFunction<T> = DEFAULT_PREDICATE_FN;
+
+    readonly filterFormControl = new FormControl();
+    readonly _filteredValues = new ReplaySubject<Array<T>>();
```

Comment:
why is this prefixed with _ ?

Reply:
filteredValues is also something that is outputtet via a get-method, but only as an observable and not as a changeable subject.